nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/manage-remote-ports/_manage-remote-ports.component-theme.scss

```diff
@@ -18,17 +18,28 @@
 @use 'sass:map';
 @use '@angular/material' as mat;
 
-@mixin nifi-theme($theme) {
+@mixin nifi-theme($theme, $canvas-theme) {
     // Get the color config from the theme.
     $color-config: mat.get-color-config($theme);
+    $canvas-color-config: mat.get-color-config($canvas-theme);
 
     // Get the color palette from the color-config.
     $primary-palette: map.get($color-config, 'primary');
+    $canvas-accent-palette: map.get($canvas-color-config, 'accent');
 
     // Get hues from palette
     $primary-palette-500: mat.get-color-from-palette($primary-palette, 500);
+    $canvas-accent-palette-A200: mat.get-color-from-palette($canvas-accent-palette, 'A200');
 
     .manage-remote-ports-header {
         color: $primary-palette-500;
     }
+
+    .manage-remote-ports-table {
+        .listing-table {
+            .fa.fa-warning {
+                color: $canvas-accent-palette-A200;
```

Comment:
@mcgilman I wasn't sure what color (if any) this icon should be? Red seems too strong and is used throughout nifi to represent an error or invalid. However, the application does not seem to mix using this `fa-warning` icon with a red color but rather these `fa-warning` icons typically have a yellow color. I am also fine if you think we just want to leave the icon with no extra colors applied and just let it be the default `.listing-table .icon` color.